shell/browser/api/electron_api_utility_process.cc

```diff
@@ -53,6 +53,25 @@ GetAllUtilityProcessWrappers() {
   return *s_all_utility_process_wrappers;
 }
 
+namespace {
+
+bool IsValidWrappable(const v8::Local<v8::Value>& obj) {
+  v8::Local<v8::Object> port = v8::Local<v8::Object>::Cast(obj);
+
+  if (!port->IsObject())
+    return false;
+
+  if (port->InternalFieldCount() != gin::kNumberOfInternalFields)
+    return false;
+
+  const auto* info = static_cast<gin::WrapperInfo*>(
+      port->GetAlignedPointerFromInternalField(gin::kWrapperInfoIndex));
+
+  return info && info->embedder == gin::kEmbedderNativeGin;
+}
```

Comment:
This looks like it's duplicating the logic of `FromV8`. See https://source.chromium.org/chromium/chromium/src/+/main:gin/wrappable.cc;l=80-99;drc=f4a00cc248dd2dc8ec8759fb51620d47b5114090 and https://source.chromium.org/chromium/chromium/src/+/main:gin/wrapper_info.cc;l=10-16;drc=f4a00cc248dd2dc8ec8759fb51620d47b5114090.

If this does in fact differ from this logic somehow, can we leave a comment about it so it doesn't later get refactored away?

Reply:
Done in a8afd88f44